### DISCUSSION TITLE
blog/istio-ambient-mode-limitations/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#353] [Announcements]
# blog/istio-ambient-mode-limitations/

深入探讨 Istio 1.22 版本中的 Ambient 模式，与传统 Sidecar 模式的对比及其限制。

https://jimmysong.io/blog/istio-ambient-mode-limitations/

**Comment by rootsongjc** [comment]
[Istio 1.23 的发布](https://istio.io/latest/news/releases/1.23.x/announcing-1.23/)解决了 Ambient mode的一些局限性：

- 在 waypoint 代理中支持 `DestinationRule`。
- 在 waypoint 和 ztunnel 中支持 DNS `ServiceEntries`。
- 支持跨命名空间共享 waypoint。
- 支持新的 `Service` 字段 `trafficDistribution`，允许保持流量在本地区域 / 地区。
- 支持双栈和 IPv6 集群。
- 为 ztunnel 提供一个新的 Grafana 仪表板。
- 一个 Helm 图表，用于一次性安装所有环境模式组件。
- 性能改进：我们的测试显示与 Istio 1.22 相比，吞吐量提高了高达 50%。
- 大量的错误修复：改善 pod 启动、支持无选择器的 Services、改善日志记录等！
